### PR TITLE
[5] Generator: Maze Walls

### LINKED ISSUE #5: Populate maze with walls
Once the initial start to finish path has been created, fill some portion of the rest of the maze with walls. Be sure to leave room for dead ends. Ideally this will be the last step in maze generation. 

- [x] Each cell that becomes a wall is colored and given a unique `textContent`
- [x] After walls are created, a single, fastest path remains
- [x] Path includes dead ends (not just a direct path)
- [x] Walls are sudo-random (random other than the rules above)

## Mockup

image

### PR DESCRIPTION
**Pull request by Thenlie**
# Generate random maze walls around path ✅ 

When the "generate maze" button is clicked, a maze is displayed with walls that increase the complexity of the solution. These walls can not be generated on the path and have a 50% chance of being generated in every other square. In my testing, this seems to be a pretty adequate way to generate walls, albeit simple. This could be expanded on later to ensure more dead ends or complex solutions. 

> NOTE: The path highlighting is no longer displayed. 

Closes #5 

## Screenshot

image

### CODE REVIEW

**Comment by Thenlie** [comment]
## 🎉 This completes the Maze Generation Component milestone! ✅ 

Once approved, if everything looks good, all focus can be put on the pathfinding algorithm and Maze Solving milestone. Work can also be done on the front end to polish things up and make the site more presentable. Issues to come!

**Review by maxh1231 — APPROVED** [review]
:)

**Pull request merged**
